setup.py

```diff
@@ -18,16 +18,12 @@
     long_description_content_type="text/markdown",
     url="",
     packages=setuptools.find_packages(),
-    install_requires=[
-        "jsonlines",
-        "regex",
-        "filetype",
-    ],
+    install_requires=["jsonlines", "regex", "filetype"],
```

Comment:
Sometimes you use multiple lines, other times you use a single line. What's your reasoning?

Reply:
Auto-formatted with [black](https://github.com/psf/black)